src/.env.sample

```diff
@@ -180,5 +180,8 @@ OBSERVATION_DEEP_LINK_REGEX= "^https:\/\/dev\.elevate-ml\.shikshalokam\.org\/vie
 # Mongo DB
 MONGODB_URL=mongodb://localhost:27017/elevate-project
 
+# detault data manager roles 
```

Comment:
@adithyadinesh0412 mention what  capability they have, like viewing dashboard or something like that